haw-pr.jenkinsfile

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2019 HAW Hamburg
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Based oin eclipse-nightly.jenkinsfile
+ */
+pipeline {
+    agent any
+
+    environment {
+        NODEJS_PATH= '/usr/bin' // '/shared/common/node-v10.15.3-linux-x64/bin'
+        YARN_PATH  = '/usr/bin' // '/shared/common/yarn/1.15.2/bin'
+        // PATH       = "${PATH}"
+                     // ":/shared/common/node-v10.15.3-linux-x64/bin"      +
+                     // ":/shared/common/java/openjdk/jdk11-x64/bin"
+        MAVEN_OPTS = '-Xmx4G'
+        JAVA_HOME  = '/usr/lib/jvm/java-11-openjdk-amd64'
+        TIMESTAMP  = new Date().format("yyyyMMddHHmm")
+   }
+
+    stages {
+        stage('Build and Test') {
+            steps {
+                echo 'Starting Xvnc'
+                wrap([$class: 'Xvnc', takeScreenshot: false, useXauthority: true]) {
+                    echo 'Building and testing, using display ' + DISPLAY
+                    // dir('n4js') {
+                        script {
+                        	def envvars = [ 
+                        		'-DDISPLAY=localhost'+DISPLAY,
+                                '-DargLine="-DDISPLAY=localhost'+DISPLAY+'"'
+                        	].join(' ')
+                            def options = [
+                                '--batch-mode',
+                                //'--quiet',
+                                '--update-snapshots',
+                                '--show-version',
+                                '-Dtycho.localArtifacts=ignore',
+                                // for adjusting the script only, renable later on:
```

Comment:
Is this comment still valid?

Reply:
I did not remove it (although I only required it for testing the script and the server) as this might be useful when updating the script or creating a new server.